### DISCUSSION TITLE
Bilingual Phrases (ie. Spanish with Interspersed English Words)

### DISCUSSION THREAD

**Discussion by kingpijamas** [deepgram/community#244] [General help] [answered]
Hey there,

We're working on a pretty unique project that needs to live-transcribe audio that's mostly in Latin American Spanish, but also has English words thrown in there from time to time. Naturally, accents will also come into play, so it's very tricky.

Could Deepgram handle this as it currently stands, or potentially in the near future? If not, we would be grateful for any suggestions or workarounds to manage this kind of scenario. We're pondering whether a custom model could be a potential solution and would love your insight on that.

Thanks in advance for your help!

Best,
JP

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @kingpijamas, my current recommendation is our Latin American Spanish model, which is available in the `general` model and `enhanced` / `base` tiers. For instance, your request can include `language=es-419&model=general&tier=enhanced`. You can check out our full [Language documentation](https://developers.deepgram.com/docs/language).

If you're interested in potentially participating in a future beta test for a relevant model, let me know! 

ETA: Spanish Nova is now available, such as `language=es-419&model=general&tier=nova`! This will be the best balance of accuracy, speed, and cost for most users.
